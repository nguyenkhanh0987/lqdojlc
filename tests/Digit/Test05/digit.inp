100
89619 95033
58854 78542
66500 76434
33772 49263
74942 90374
3706 41272
89121 90155
70556 73583
24644 98068
79086 79109
4141 41566
71183 91515
89960 92377
67780 73390
5134 78915
30154 64359
4123 52497
21981 64421
81670 82715
9704 41108
62741 65472
60526 68737
98078 98857
94712 95049
79003 96117
57956 67813
99067 99138
10768 18576
40522 91356
30498 85509
25059 73370
53777 72851
62828 65223
34213 62616
65129 88131
69256 93391
626 49881
1152 20788
36700 73401
84975 99895
13824 23343
80556 99463
23465 36920
96764 96969
56259 67205
12082 40578
69537 92423
71079 78298
41566 84690
17681 70440
35021 74507
12481 99862
43072 99695
72994 83835
59106 90781
78141 97506
71481 75941
76703 98929
29408 30527
26158 89732
57331 70051
64553 68703
26774 94557
20379 40683
31110 59631
58680 65552
98099 98650
59193 63979
56404 63249
49668 99148
41135 77204
67728 98717
63184 81749
82772 94153
47526 93964
88881 92520
49434 53521
42205 99763
80560 97366
40308 75010
28024 88427
61272 73426
58324 58773
2922 39201
10978 81455
88092 95636
64443 99963
96246 99302
99304 99591
11075 96144
43232 49521
29634 95900
63541 99412
37834 42307
6204 19921
10348 75529
31709 36498
71289 91165
61896 76313
94387 99358
