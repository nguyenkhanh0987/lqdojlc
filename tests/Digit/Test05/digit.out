23
1246
598
2146
807
6767
84
30
6297
7
6711
862
1
357
9513
3485
7852
5060
4
5109
125
491
51
0
546
750
13
1465
3548
4622
5316
1489
105
3166
1255
971
8266
4107
3481
579
1810
594
2036
0
889
4851
909
317
3190
6437
3812
8577
3716
505
1721
688
97
866
177
5933
1133
395
5813
3033
3322
393
42
201
611
2673
2877
1356
1142
366
2943
113
194
3742
454
2742
5396
701
70
6695
8143
209
1700
171
23
8531
930
5062
1722
615
3106
7755
539
861
816
208
